inductiva/tasks/streams.py

```diff
@@ -10,7 +10,7 @@
 from inductiva import constants
 
 logger = logging.getLogger("websocket")
-logger.setLevel(logging.INFO)
+logger.setLevel(logging.DEBUG)
```

Comment:
Why the change here?

Reply:
The reason is to reduce the ping/pong messages on reconnect, but Sérgio will address this soon.